packages/create-gator-app/templates/nextjs/advanced-permissions-starter/src/components/RedeemPermissionButton.tsx

```diff
@@ -52,7 +46,7 @@ export default function RedeemPermissionButton() {
         return;
       }
 
-      const { fast: fee } = await pimlicoClient.getUserOperationGasPrice();
+      const { fast: fee } = await pimlicoClient(chain.id).getUserOperationGasPrice();
```

Comment:
this (and the other clients) is a little confusing because we expect `pimlicoClient` or `bundlerClient` to _be_ that client.

Can we name this `pimlicoClientFactory` / `bundlerClientFactory` etc?